src/commands/ontology.js

```diff
@@ -111,6 +113,36 @@ class VocabLoad extends Command {
   }
 }
 
+class VocabExport extends Command {
+  static get action() { return VOCAB.EXPORT }
+
+  *exec() {
+    const { payload } = this.action
+    const [vocab, classes, props, types] =
+          yield select(state => [
+            values(pick(state.ontology.vocab, payload)),
+            state.ontology.class,
+            state.ontology.props,
+            state.ontology.type
+          ])
+
+    const path = yield call(dialog.save.vocab)
+    if (!path) return
+
+    const data = yield call(toN3, vocab[0], classes, props, types)
+
+    try {
+      yield call(async () => { write(path, await data) })
+    } catch (error) {
+      warn(`Failed to export "${vocab.id}": ${error.message}`)
```

Comment:
I'm just converting error reporting to `warn(message, { error })` (so no need to the extra debug/verbose stack trace).